lib/src/err/mod.rs

```diff
@@ -43,8 +43,8 @@ pub enum Error {
 	RetryWithId(Thing),
 
 	/// The database encountered unreachable logic
-	#[error("The database encountered unreachable logic")]
-	Unreachable,
+	#[error("The database encountered unreachable logic: {0}")]
+	Unreachable(String),
```

Comment:
This is a breaking change. Unfortunately we can't make it in v1 because this enum is public. If you really want to, you should add a new variant instead.

Reply:
I will add a new variant in that case 👍